### PR TITLE
Updated Italian translations

### PR DESCRIPTION
**Pull request by billmn**
Aligned to the latest version of Laravel

### CODE REVIEW

File: resources/lang/it/validation.php [hunk 1]
```diff
@@ -22,50 +22,54 @@
...
-    'accepted'   => ':attribute deve essere accettato.',
-    'active_url' => ':attribute non è un URL valido.',
-    'after'      => ':attribute deve essere una data successiva al :date.',
-    'alpha'      => ':attribute può contenere solo lettere.',
-    'alpha_dash' => ':attribute può contenere solo lettere, numeri e trattini.',
-    'alpha_num'  => ':attribute può contenere solo lettere e numeri.',
-    'array'      => ':attribute deve essere un array.',
-    'before'     => ':attribute deve essere una data precedente al :date.',
-    'between'    => [
+    'accepted'             => ':attribute deve essere accettato.',
```
Comment (GrahamCampbell) [inline resources/lang/it/validation.php]: Please don't add extra indentation.

Comment (GrahamCampbell) [inline resources/lang/it/validation.php]: Please indent to the minimal level.

Comment (billmn) [inline resources/lang/it/validation.php]: Ok, I've used current Laravel indentation for this file: https://raw.githubusercontent.com/laravel/laravel/master/resources/lang/en/validation.php

I reduce it.

Comment (billmn) [inline resources/lang/it/validation.php]: Also remove extra indentation from EN lang file or do you want to keep indentation like original Laravel's file?

Comment (GrahamCampbell) [inline resources/lang/it/validation.php]: Don't keep it like Laravel. Just align it at the lowest level possible.

Comment (GrahamCampbell) [inline resources/lang/it/validation.php]: StyleCI doesn't yet fix this, but will soon.

Comment (billmn) [inline resources/lang/it/validation.php]: Cool!

**Pull request merged**